<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Recherche client</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineNom"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="linePrenom"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Date de rendez-vous début / fin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateRdvDebut">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateRdvFin">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Numéro d'identification</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineNumeroID"/>
          </item>
          <item>
           <widget class="QPushButton" name="BouttonRechercherClient">
            <property name="text">
             <string>Rechercher</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableViewClient"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Liste du personnel</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeView" name="treeViewPersonnel"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonModifierPersonnel">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSupprimerPersonnel">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Liste des clients</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>511</width>
          <height>511</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTableView" name="tableViewClient_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonModifierClient">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSupprimerClient">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Planification</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>491</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableView" name="tableViewPlannification"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionclient"/>
     <addaction name="actionPersonnel"/>
    </widget>
    <addaction name="actionQuitter"/>
    <addaction name="menuAjouter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menufichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionclient"/>
   <addaction name="actionPersonnel"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionclient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionPersonnel">
   <property name="text">
    <string>Personnel</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
